korangar/src/graphics/renderers/picker/selector/selector.wgsl

```diff
@@ -9,6 +9,7 @@ var<push_constant> constants: Constants;
 @compute
 @workgroup_size(1)
 fn cs_main() {
-    let texel_value = textureLoad(texture, constants.pointer_position, 0);
-    buffer[0] = texel_value.x;
+    let texel_value = textureLoad(texture, constants.pointer_position, 0).rg;
+    buffer[0] = texel_value.r;
+    buffer[1] = texel_value.g;
```

Comment:
Can we make the `buffer` an `array<u64>`?

It feels a bit wrong that the buffer is of type `u64` but the shader treats it as `u32`

Reply:
WGSL has no u64.

https://www.w3.org/TR/WGSL/#plain-types-section